jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -226,6 +248,8 @@ def mos_spec1d_parser(app, data_obj, data_labels=None,
         _add_to_table(app, data_labels, '1D Spectra',
                       table_viewer_reference_name=table_viewer_reference_name)
 
+    return len(data_obj)
```

Comment:
If multiple objects were loaded from the same file, we need to know how many times to repeat some information in the table data.

Reply:
This is being treated as an error for the snackbar because of https://github.com/spacetelescope/jdaviz/blob/5232ca0af9bd0425eb07647046babf8ef09b9a75/jdaviz/app.py#L530-L538

Any ideas what we should change to prevent tha twhile keeping the information needed for the mos table?

Reply:
Wut.... I didn't even know about this behavior. Should we just remove it? Parser can emit their own messages already.

Reply:
If no parser currently relies on returning something other than None to provide the error message, then sure!  Otherwise I think we could easily migrate anything to emitting their own snackbar error instead of making this assumption.

Reply:
Created #1888 to get this in a follow-up.  Until then this just results in an error snackbar message with a number in it, which might cause a bit of confusion, but isn't critical.